front/app/containers/MainHeader/Components/DesktopNavItems/AdminPublicationsNavbarItem.tsx

```diff
@@ -159,6 +159,12 @@ const AdminPublicationsNavbarItem = ({
 
   const urlSegments = location.pathname.replace(/^\/+/g, '').split('/');
   const secondUrlSegment = urlSegments[1];
+  const thirdUrlSegment = urlSegments[2];
```

Comment:
I'm having trouble understanding what these `urlSegments` represent. Is this playing a role in hiding the project items if they are not visible?

Reply:
Basically, it is checking if there is something after `/projects/` and in this way it determines whether the `All projects` navbar item is active or not. I introduced this mechanism because if a project is pinned both the project item and `All projects` item in the navigation bar were active and this looked confusing.